Backpack/BarChart/Classes/BarChartCollectionView.swift

```diff
@@ -99,8 +102,18 @@ public final class BPKBarChartCollectionView: UICollectionView {
                 else {
                     return
             }
-            let selectedBarTopPosition: CGFloat = selectedCell.barChartBar.barTopPosition
-            selectedMarkerBottomConstraint.constant = selectedBarTopPosition
+
+            if selectedCell.barChartBar.fillValue == nil {
+                selectedBarOpacity = 0
+            } else {
+                selectedBarTopPosition = selectedCell.barChartBar.barTopPosition
+                selectedBarOpacity = 1
+            }
+        }
+        UIView.animate(withDuration: BPKDuration.animationDurationSm) {
+            self.selectedMarkerBottomConstraint.constant = selectedBarTopPosition
```

Comment:
`self.` is necessary here as the scope cannot be inferred inside an animation code block